esx_service/vmci/vmci_client.c

```diff
@@ -360,7 +360,7 @@ Vmci_GetReply(int port, const char* json_request, const char* be_name,
       return CONN_FAILURE;
    }
 
-   req.mlen = strnlen(json_request, MAXBUF) + 1;
+   req.mlen = strnlen(json_request, MAXBUF-1) + 1;
```

Comment:
This fix is from reviewing the code, the original panic isn't reproducible anymore with the GO env. upgrade.